lib/libzrepl/data_conn.c

```diff
@@ -1834,10 +1836,14 @@ uzfs_zvol_io_ack_sender(void *arg)
 				}
 			} else if (zio_cmd->hdr.opcode == ZVOL_OPCODE_SYNC) {
 				atomic_inc_64(&zinfo->sync_req_ack_cnt);
-				atomic_add_64(&zinfo->sync_latency,
-				    gethrtime() - zio_cmd->io_start_time);
+				latency = (gethrtime() -
```

Comment:
please add latency for  READ and WRITE IOs also.

Reply:
latency is already there for READ and WRITE IOs

Reply:
@pawanpraka1 ^^